pkg/recordset/recordset.go

```diff
@@ -571,6 +578,9 @@ func getManagedRecordSets(clusterID, baseDomain string) []string {
 		fmt.Sprintf("\\052.%s.%s.", clusterID, baseDomain), // \\052 - `*` wildcard record
 		fmt.Sprintf("api.%s.%s.", clusterID, baseDomain),
 		fmt.Sprintf("etcd.%s.%s.", clusterID, baseDomain),
+		fmt.Sprintf("etcd1.%s.%s.", clusterID, baseDomain),
+		fmt.Sprintf("etcd2.%s.%s.", clusterID, baseDomain),
+		fmt.Sprintf("etcd3.%s.%s.", clusterID, baseDomain),
```

Comment:
What is our idea with managing this over time? Is it that at some point the `etcd.` domain is not necessary anymore? How do we migrate away from it? Is it just that cluster upgrade and then do not use it anymore at some point? I am asking because my goal would be to use `etcd1.` even for single master clusters.

Reply:
that does not work, you have an ELB that has DNS name `etcd` and it load balance between all nodes  - `etcd1` in case of single master and `etcd1,etcd2,etcd3` in case of HA masters.

Its not redundant name

Reply:
I see. That is good to know. That is cool then. Thank you for the explanation. 🤗